pilot/pkg/model/push_context.go

```diff
@@ -1149,6 +1150,7 @@ func (ps *PushContext) initVirtualServices(env *Environment) error {
 					ps.privateVirtualServicesByNamespaceAndGateway[ns][gw] = append(ps.privateVirtualServicesByNamespaceAndGateway[ns][gw], virtualService)
 				}
 			} else if ps.defaultVirtualServiceExportTo[visibility.Public] {
+				fmt.Println("---gwname ", gwNames)
```

Comment:
can the print here be remove or replace with log?

Reply:
I should remove, it was used to debug ut failure